src/com/google/javascript/rhino/JSDocInfo.java

```diff
@@ -223,6 +224,21 @@ private int getMaskForBitIndex(int bitIndex) {
    * with a string.
    */
   public static class StringPosition extends SourcePosition<String> {
+    static boolean areEquivalent(StringPosition p1, StringPosition p2) {
```

Comment:
That would work for SourcePosition<String> but not SourcePosition<Node>. We have to call node1.isEquivalentTo(node2) instead of equals() on Node because the source files are different :(